content/en/community/end-user/interviews-feedback.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Interviews or Feedback Sessions
+linkTitle: Interviews or Feedback Sessions
+Description: >-
+ Help improve OpenTelemetry - share your feedback directly with us!
+---
+
+One of the core functions of the OpenTelemetry End User Working Group is 
+to improve the project by gathering feedback from end users and sharing
+them with the appropriate SIGs to help drive prioritization of 
+improvements and changes to the project. 
+
+Direct interview or feedback sessions between an organization and the 
+OpenTelemetry Community is one such resource. In these hour-long sessions, 
+we learn about the organization's OTel adoption and implementation, with 
+particular interest in the following:
+
+* Special use cases 
+* Challenges with implementation -- what went well? What needs improvement? 
+  These can be anything, from instrumentation to documentation. 
+* Challenges with adoption within the organization -- how can the 
+  Community help you drive adoption with your teams? 
+
```

Comment:
Worth mentioning that we will share feedback with the appropriate SIGs so that they can be actioned?

Reply:
I added this below:

"We will then take the feedback you shared and turn them into actionable tasks 
for the relevant SIGs, as well as provide context and insight for you during
the session as appropriate."